ui/app/styles/helper-classes/spacing.scss

```diff
@@ -87,6 +87,10 @@
   margin-bottom: -$spacing-m;
 }
 
+.has-bottom-margin-xxs {
+  margin-bottom: $spacing-xxs !important;
```

Comment:
Did you test without the important? I know we have importants everywhere in our helpers, and even bulma had importants, but the feedback I got on the bulma work was do add them if we don't need them.